chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -101,6 +99,7 @@ export enum TelemetryEvent {
     WidgetLoadComplete = "WidgetLoadComplete",
     WidgetLoadFailed = "WidgetLoadFailed",
     StartChatMethodException = "StartChatMethodException",
+    CloseChatCall = "CloseChatCall",
```

Comment:
Please log all telemetry changes in Telemetry.md

Reply:
Updated Telemetry file